iaso/models/org_unit.py

```diff
@@ -552,3 +552,99 @@ class OrgUnitReferenceInstance(models.Model):
     class Meta:
         # Only one `instance` by pair of org_unit/form.
         unique_together = ("org_unit", "form")
+
+
+class OrgUnitChangeRequest(models.Model):
+    """
+    A request to change an OrgUnit.
+
+    It can also be a request to create an OrgUnit. In this case the OrgUnit
+    already exists in DB with `OrgUnit.validation_status == VALIDATION_NEW`.
+    The change request will change `validation_status` to either
+    `VALIDATION_REJECTED` or `VALIDATION_VALID`.
+    """
+
+    class Statuses(models.TextChoices):
+        NEW = "new", _("New")
+        REJECTED = "rejected", _("Rejected")
+        VALIDATED = "validated", _("Validated")
+
+    class ChangeRequestFields(models.TextChoices):
+        """
+        Fields for which a change can be requested.
+        """
+
+        PARENT = "parent"
+        NAME = "name"
+        ORG_UNIT_TYPE = "org_unit_type"
+        GROUPS = "groups"
+        LOCATION = "location"
+        INSTANCES = "instances"
+
+    org_unit = models.ForeignKey("OrgUnit", on_delete=models.CASCADE)
+    status = models.CharField(choices=Statuses.choices, default=Statuses.NEW, max_length=40)
+
+    # Metadata.
+
+    created_at = models.DateTimeField(default=timezone.now)
+    created_by = models.ForeignKey(
+        User, null=True, blank=True, on_delete=models.SET_NULL, related_name="org_unit_change_created_set"
+    )
+    updated_at = models.DateTimeField(blank=True, null=True)
+    updated_by = models.ForeignKey(
+        User, null=True, blank=True, on_delete=models.SET_NULL, related_name="org_unit_change_updated_set"
+    )
+    reviewed_at = models.DateTimeField(blank=True, null=True)
+    reviewed_by = models.ForeignKey(
+        User, null=True, blank=True, on_delete=models.SET_NULL, related_name="org_unit_change_reviewed_set"
+    )
+    rejection_comment = models.CharField(max_length=255, blank=True)
+
+    # Fields for which a change can be requested.
+
+    parent = models.ForeignKey(
+        "OrgUnit", null=True, blank=True, on_delete=models.CASCADE, related_name="org_unit_change_parents_set"
+    )
+    name = models.CharField(max_length=255, blank=True)
+    org_unit_type = models.ForeignKey(OrgUnitType, on_delete=models.CASCADE, null=True, blank=True)
+    groups = models.ManyToManyField("Group", blank=True)
+    location = PointField(null=True, blank=True, geography=True, dim=3, srid=4326)
+    # `accuracy` is only used to help decision-making during validation: is the accuracy good
+    # enough to change the location? The field doesn't exist on `OrgUnit`.
+    accuracy = models.DecimalField(decimal_places=2, max_digits=7, blank=True, null=True)
+    # `instances` is a list to be defined as new "reference instances".
+    instances = models.ManyToManyField("Instance", blank=True)
```

Comment:
maybe call it differently, then? reference_instances ?

Reply:
Yes you're right!

@bmonjoie is [renaming `instances`](https://github.com/BLSQ/iaso/pull/795/files#diff-fe3802363f51a9c9a55bd494caa6709b2f48a3d0c0a5dbcdcaf8c4e0a50c8e4dR129) to `reference_instances ` OK for you?